browser/components/Main.js

```diff
@@ -1,55 +1,59 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import axios from 'axios';
 
-import StudentList from './StudentList.js' 
-import SingleStudent from './SingleStudent.js' 
+import StudentList from './StudentList.js';
+import SingleStudent from './SingleStudent.js';
 
 export default class Main extends Component {
-    constructor(props){
-        super(props)
-        this.state = {
-            students: [],
-            selectedStudent : {}
-        }
+  constructor(props) {
+    super(props);
+    this.state = {
+      students: [],
+      selectedStudent: {}
+    };
+  }
 
-        this.selectStudent = this.selectStudent.bind(this)
-    }
+  componentDidMount() {
+    this.getStudents();
+  }
 
-    componentDidMount(){
-        this.getStudents()
+  getStudents = async () => {
```

Comment:
Slacked you on this one.

Reply:
For the record, the goal is to update all curricular repos so that arrow functions work throughout (issue is older versions of Babel). We (Instructors) will cover this in our lectures.